shared/ui-components/src/locales/general.json

```diff
@@ -150,7 +151,10 @@
       "provideAMailingAddress": "Provide an address where you can receive updates and materials about your application.",
       "doYouWorkIn": "Do you work in <REGION>?",
       "doYouWorkInDescription": "TBD",
-      "workAddress": "Work Address"
+      "workAddress": "Work Address",
+      "yes": "Yes",
+      "no": "No",
```

Comment:
Yes and No are already in the "t" category, so let's not add them a second time. (Actually a third time, I'm noticing another duplication in the JSON.)